src/sql/azdata.proposed.d.ts

```diff
@@ -1741,9 +1741,181 @@ declare module 'azdata' {
 		ownerUri: string;
 	}
 
+	export interface SchemaCompareParams {
+		sourceEndpointInfo: SchemaCompareEndpointInfo;
+		targetEndpointInfo: SchemaCompareEndpointInfo;
+		taskExecutionMode: TaskExecutionMode;
+		deploymentOptions: DeploymentOptions;
+	}
+
+	export interface SchemaCompareGenerateScriptParams {
+		operationId: string;
+		targetDatabaseName: string;
+		scriptFilePath: string;
+		taskExecutionMode: TaskExecutionMode;
+	}
+
+	export interface SchemaCompareGetOptionsParams {
```

Comment:
this shouldn't be needed here since it's also defined in contracts.ts

Reply:
same